docs/index.md

```diff
@@ -4,46 +4,67 @@
 
 ### How to document
 
-Before commanding everyone to start writing, I would like to list why documenting is important.
+There are very few docs that make us understand how, why and what to document. One of it is [The documentation system — Documentation system documentation](https://documentation.divio.com/)
+
+| ![overview](assets/index_1.png) |
+|:--:|
+| [*Source*](https://documentation.divio.com/) |
+
+A very good guide describing 'why documentations are required in coding?' can be found here:
+[A beginner’s guide to writing documentation — Write the Docs](https://www.writethedocs.org/guide/writing/beginners-guide-to-docs/).
+
+An insightful guide for why and how to document your technical writings can be found here: [What to write - Beginners Guide to docs - writethedocs](https://www.writethedocs.org/guide/writing/beginners-guide-to-docs/#what-to-write).
 
 ### What to document
 
 1. Any novel procedure/technique used apart of previously documented work in the same field.
-2. Usage of new equipments/tools/softwares/boards that were not previously used and its usage had drastically improved the working of the vehicle.
-3. A new idea/concept (which has already proven by other teams but couldn't be completed under the team members tenure in the club and he/she has to transfer this idea to the juniors) that has been used for improving the efficiency and design of the vehicle.
-4. Any undocumented but significant basic procedure that is used in the working of the vehicle.
-5. Any undocumented work that was previously left incomplete/neglected by the seniors.
+2. Usage of new types of equipment/tools/software/boards that were not previously used and its usage had drastically improved the working of the vehicle.
+3. A new idea/concept that has been used for improving the efficiency and design of the vehicle. This idea may have been already proven by other teams but couldn't be completed under the team member's tenure in the club and he/she has to transfer this idea to the juniors.
+4. Any undocumented (significant) procedure that is used in the working of the vehicle.
+5. Any undocumented work that was previously left incomplete/missed by the seniors.
+6. Comparative studies - Survey of pros and cons of techniques or hardware/pieces of equipment, which will help in buying the best and efficient one.
 
 ### What NOT to document
 
-1. Those which do not have concrete evidence for the result (The student can better read plenty of research papers without a concrete goal).
-2. Comparative studies - This is not journal publishing stories. Here students must benefit from the technique and not stay confused while choosing one.
+1. Those which do not have concrete evidence for the result.
+2. Redundant information that has already been done by previous members.
+
+### Format of docs
+
+The report must be in the format:
+
+```title:``` Title of the report
```

Comment:
nit pick: Single back ticks also work the same way for a single linke code block.